server/controllers/index.js

```diff
@@ -182,12 +182,20 @@ export const explore = async(ctx, next) => {
   };
 
   const aDropInTheOceanPromoList = PromoListFactory.fromSeries(aDropInTheOceanSeries);
+  const digitalStories = commissionedSeries.filter(s => s.commissionedLength);
```

Comment:
I'm not sure if this is the best way to find the most recent digital story – sorting the items of all the stories by date, taking the most recent one, then finding the story that _that_ one lives in. Other/better ways of doing this welcomed.